config.xml

```diff
@@ -265,6 +265,7 @@
 <!-- ALP -->
   <doc cat="alp" doc="alp-dolomite" branches="main">The SUSE ALP Dolomite Guide</doc>
   <doc cat="alp" doc="cockpit-alp-dolomite" branches="main">The SUSE ALP Dolomite Cockpit Guide</doc>
+  <doc cat="alp" doc="deploying-alp-dolomite-interactively-using-agama" branches="main">The SUSE ALP Dolomite Cockpit Guide</doc>
```

Comment:
The title is wrong, shouldn't it be "Deploying ALP Dolomite interactively using Agama"?